aips/passkeys.md

```diff
@@ -0,0 +1,412 @@
+
+1. **User-friendliness:**
+   1. WebAuthn credential registration (private key creation) and transaction signing can be done simply via device biometrics
+   2. Enables users to interact with dapps via their Passkey accounts, without having to install a mobile or extension wallet: i.e., a **headless wallet experience**.
+   3. By storing the private key securely on the device instead of the browser, passkeys eliminate the need for setting up a wallet password, traditionally required for encrypting private keys in the browser.
+   4. On certain operating systems, the passkey is backed up and syncs seamlessly with multiple devices (see [backup eligibility](#backup-eligibility)).
+
+2. **Security:**
+   1. With passkeys, the private key is stored securely on the device instead of the browser, alleviating the need to store any sensitive material in browser storage where it could be potentially be accessed by a malicious party either through physical access on the device or supply chain attacks via a malicious dependency.
+   2. By default, passkeys provide a consent-driven signing experience and prompt the user to authenticate every time they sign, similar to Apple Pay or Google Pay.
+   3. Passkeys are bound to a relying party (e.g., a website) and are not phishable.
+
+## Alternative solutions
+
+See the section on **Alternative Solutions** in [AIP-61](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-61.md#alternative-solutions) for a more in-depth exploration into the respective tradeoffs of multi-party computation (MPC), hardware security modules (HSMs), and OpenID blockchain (OIDB) accounts.
+
+### OIDB Accounts
+
+OIDB accounts present a unique and novel way to generate an on-chain account with an OpenID account. This is highly desirable as OIDB accounts are accessible across all devices with browsers. Additionally, most people have an account associated with one or more OIDC providers (e.g., Google) and are familiar with OAuth login flows. Lastly, recoverability is not limited to certain operating systems in the same way that passkeys are.
+
+On the other hand, there are tradeoffs to consider as well:
+
+1. There are centralization and liveness concerns both on the OIDC provider and the services that OIDB accounts require to function in a privacy preserving manner (e.g., pepper service, proving service) respectively.
+2. Unless the ephemeral secret key ($\mathsf{esk}$) associated with the OIDB account of the user is encrypted (i.e., with a password) or is a passkey itself, the plaintext ephemeral secret key is still available on the browser, leaving it potentially vulnerable to a malicious actor (but only during its short expiration window). Passkey private keys, on the other hand, are stored on the device when the passkey is backed up, often within a secure element on the device and not on the browser.
+
+## Specification
```

Comment:
Could we have a very brief overview of the end to end flow?

* User visits site and is asked to register a passkey for the site
* User generates an aptos address from the passkey using ...
* the user transfers funds to that account or the dapp offers sponsored transactions (AIP-52) to create the account
* the user initiates an action within a dapp
* the dapp asks for permissions to submit a transaction
* the system prompt asks for permissions to use the passkey
* the passkey signs the user transaction using webauthn

Reply:
Added